src/mono/mono/mini/mini-llvm.c

```diff
@@ -4572,14 +4572,19 @@ emit_landing_pad (EmitContext *ctx, int group_index, int group_size)
 }
 
 static LLVMValueRef
-create_const_vector_i32 (const int *mask, int count)
+create_const_vector (LLVMTypeRef t, const int *vals, int count)
 {
-	LLVMValueRef *llvm_mask = g_new (LLVMValueRef, count);
+	g_assert (count <= 16);
```

Comment:
Why 16? `Vector256<T>` supports up to 32...

Reply:
@tannergooding we don't support Vector256 yet (and I am not sure we will any time soon)

Reply:
Mono doesn't support AVX vectors or AVX intrinsics right now. This function is only ever used by other procedures that translate our internal IR to LLVM IR, and we only ever create constant vectors of up to length 16 while doing this.